corehq/apps/cloudcare/static/cloudcare/js/debugger/debugger.js

```diff
@@ -1,9 +1,25 @@
 'use strict';
-/* globals ace, Clipboard */
-hqDefine('cloudcare/js/debugger/debugger', function () {
-    var kissmetrics = hqImport("analytix/js/kissmetrix"),
-        readableForm = hqImport("reports/js/readable_form");
-
+hqDefine('cloudcare/js/debugger/debugger', [
+    'jquery',
+    'knockout',
+    'underscore',
+    'clipboard/dist/clipboard',
+    'ace-builds/src-min-noconflict/ace',
+    'analytix/js/kissmetrix',
+    'reports/js/readable_form',
+    'hqwebapp/js/atwho',    // $.atwho
```

Comment:
is the comment for documentation? Because I don't understand it.

Reply:
yeah, I've gotten in the habit of commenting on dependencies that don't map to a named module, usually because they add a jquery plugin or knockout bindings. Otherwise it's hard to know why a dependency is included and when/if it can be removed.  Granted, this one is pretty obvious, it's more helpful for dependencies like jquery ui and hqwebapp knockout_bindings that contain multiple features.